scripts/patch-apply-gen-test-data.rs

```diff
@@ -0,0 +1,196 @@
+// Release build is required. See https://github.com/seanmonstar/reqwest/issues/1017.
+use std::{
+    collections::HashMap,
+    default::Default,
+    io::{BufReader, Read},
+    path::PathBuf,
+};
+
+use anyhow::Context;
+use http_range_client::HttpReader;
+use rattler_conda_types::{Channel, MatchSpec, ParseStrictness, Platform, RepoDataRecord};
+use rattler_package_streaming::seek::stream_conda_info;
+use rattler_repodata_gateway::{Gateway, fetch, sparse::SparseRepoData};
+use reqwest::Client;
+use reqwest_middleware::ClientWithMiddleware;
+use tokio::task::JoinSet;
+use url::Url;
+
+const OUTPUT_PATH: &str = "test-data/conda_forge/recipes/";
+
+// Overview:
+// 1. Get repodata for conda-forge linux-64.
```

Comment:
Interesting approach! I think I would have checked if Github has some API and tried to extract them from the feedstocks. But this obviously works nicely as well!